tools/buildsys/src/manifest.rs

```diff
@@ -80,43 +80,6 @@ naming.
 package-name = "better.name"
 ```
 
-`variant-sensitive` lets you specify whether the package should be rebuilt when
-building a new variant, and defaults to false; set it to true if a package is
-using the variant to affect its build process.
-
-```ignore
-[package.metadata.build-package]
-variant-sensitive = true
```

Comment:
We'll need at least the boolean form of variant sensitivity to remain until none of the programs in `os` are conditionally compiled by way of their build script.

Otherwise what will happen is we'll build all those packages once, with the active variant from the environment, and we won't rebuild them again when the variant changes. So everything that still builds in a settings model will only be built for the first variant.

Reply:
Check my latest diff. I added a "DEPRECATED" feature flag. As it turns out, `os` is the only remaining package using `variant-sensitive`! So in the `os` `build.rs` I am emitting a `DEPRECATED` feature flag to sneak us by for a while longer while ensuring that nobody else uses `variant-sensitive`.